pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -65,6 +65,7 @@ private static string GetPathToFile(string ext)
         public static readonly Func<string> PATH_TO_TSV = () => GetPathToFile(TextUtil.EXT_TSV);
         public static readonly Func<string> PATH_TO_IRTDB = () => GetPathToFile(IrtDb.EXT);
         public static readonly Func<string> PATH_TO_BLIB = () => GetPathToFile(BiblioSpecLiteSpec.EXT);
+        public static readonly Func<string> PATH_TO_MPROPHET_CSV = () => GetPathToFile(TextUtil.EXT_CSV);
```

Comment:
Let's just use PATH_TO_CSV and not create another CSV file type with the same text.